instrumentation/active_job/test/instrumentation/active_job/patches/active_job_callbacks_test.rb

```diff
@@ -179,22 +169,16 @@
         TestJob.perform_later
 
         [send_span, process_span].each do |span|
-          _(span.attributes['messaging.system']).must_equal('inline')
+          _(span.attributes['messaging.system']).must_equal('async')
```

Comment:
I think we should do 

```ruby
      ::ActiveJob::Base.queue_adapter.shutdown rescue nil
      ::ActiveJob::Base.queue_adapter = :inline
```

here to retain the test's meaning.

Reply:
uggg I got too aggressive here. I will roll it back!

Reply:
Done!